### PR TITLE
Add `Zoom to Geometry` icon beside vector features (relation table)

### LINKED ISSUE #520: Add zoom relation feature on relation table when relation is between vector layer
### Checklist

- [X] I've searched through the [current issues](https://github.com/g3w-suite/g3w-client/issues) to make sure this feature hasn't been requested already.

### Motivation

It useful know which and where is relation vector feature

### Suggested solution

Add marker icon on table relation feature to zoom to relation feature

### Alternatives considered

_No response_

### PR DESCRIPTION
**Pull request by volterra79**
Closes: #520 

## Before

Screenshot from 2023-11-14 17-05-57

## After

Screenshot from 2023-11-14 15-44-09

### CODE REVIEW

**Comment by Raruto** [comment]
> ## Sample [project.zip](https://github.com/g3w-suite/g3w-client/files/13350015/project.zip)

@volterra79 I am unable to open that project:

image


 [g3wsdk.info]

- g3w-admin: __3.7.dev-20231115083136__
- g3w-client: __3.9.0-alpha.2.0__
- browser: __Firefox 119.0__
- operating system: __Windows 11 64-bit__

**Comment by Raruto** [comment]
@volterra79 do you get the same error? (I guess because there are 3 `<td>` but only 2 `<th>` elements)

**NB:** editing is disabled on all layers

image


image

image

**Comment by volterra79** [comment]
@Raruto fixd by https://github.com/g3w-suite/g3w-client/pull/521/commits/9e9f50b651a9e3ee291a7bccbe2235bdec3a9bfe

File: src/components/Relation.vue [hunk 1]
```diff
@@ -477,18 +527,101 @@ export default {
...
+  .query-relation.mobile > .header > .g3w-long-text {
+    font-size: 1em;
+  }
+  .query-relation.mobile #relationtable_filter.dataTables_filter {
+    display: flex !important;
+    justify-content: flex-end !important;
+  }
+  .query-relation.mobile .relations-download {
+    display: flex;
+  }
```
Comment (Raruto) [inline src/components/Relation.vue]: @volterra79 they appear unused to me:

```suggestion
```

Comment (volterra79) [reply src/components/Relation.vue]: @Raruto you can remove it

**Review by Raruto — APPROVED** [review]

**Pull request merged**